### ISSUE TITLE
syntax error, unexpected tLABEL

### ISSUE THREAD

**Issue by mattqs** [mosuka/logstash-output-solr#1]
I'm getting error:

```
SyntaxError: /opt/logstash/vendor/bundle/jruby/1.9/gems/rsolr-cloud-1.0.0/lib/rsolr/cloud/connection.rb:32: syntax error, unexpected tLABEL
      def select_node(collection, leader_only: false)
                                              ^
     require at org/jruby/RubyKernel.java:1040
     require at /opt/logstash/vendor/bundle/jruby/1.9/gems/polyglot-0.3.5/lib/polyglot.rb:65
      (root) at /opt/logstash/vendor/bundle/jruby/1.9/gems/rsolr-cloud-1.0.0/lib/rsolr/cloud.rb:1
     require at org/jruby/RubyKernel.java:1040
     require at /opt/logstash/vendor/bundle/jruby/1.9/gems/polyglot-0.3.5/lib/polyglot.rb:65
      (root) at /opt/logstash/vendor/bundle/jruby/1.9/gems/rsolr-cloud-1.0.0/lib/rsolr/cloud.rb:2
     require at org/jruby/RubyKernel.java:1040
     require at /opt/logstash/vendor/bundle/jruby/1.9/gems/polyglot-0.3.5/lib/polyglot.rb:65
      (root) at /opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-output-solr-0.2.0/lib/logstash/outputs/solr.rb:1
      (root) at /opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-output-solr-0.2.0/lib/logstash/outputs/solr.rb:9
      (root) at /opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.2.0-java/lib/logstash/plugin.rb:1
        eval at org/jruby/RubyKernel.java:1079
      lookup at /opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.2.0-java/lib/logstash/plugin.rb:80
      plugin at /opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.2.0-java/lib/logstash/pipeline.rb:385
  initialize at (eval):15
        call at org/jruby/RubyProc.java:281
  initialize at /opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.2.0-java/lib/logstash/pipeline.rb:57
        call at org/jruby/RubyProc.java:281
     execute at /opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-core-2.2.0-java/lib/logstash/agent.rb:162
```

I assume that https://github.com/enigmo/rsolr-cloud/pull/3 will fix this. Is there a workaround (I'm new to ruby) or do I have to wait until the new release?

**Comment by mosuka** [comment]
Hi @mattqs ,

If you can not wait to be released a new version, you will need to modify the following installed file directly.
/opt/logstash/vendor/bundle/jruby/1.9/gems/rsolr-cloud-1.0.0/lib/rsolr/cloud/connection.rb

How to moifiy the code, please refer to the following URL.
https://github.com/enigmo/rsolr-cloud/pull/3/files

Thanks,

**Comment by mattqs** [comment]
Hi @mosuka,

thank you, this information was very useful.

**Comment by mosuka** [comment]
Hi @mattqs ,

I have released logstash-output-solr 0.3.0. Please use new one.

Thanks,
